common/djangoapps/student/views.py

```diff
@@ -2220,14 +2225,16 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
         )
 
     if request.method == 'POST':
-        password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
+        password1 = request.POST['new_password1']
+        password2 = request.POST['new_password2']
+        is_password_valid, password_err_msg = validate_password(user, password1, password2)
+
         if not is_password_valid:
             # We have a password reset attempt which violates some security
             # policy. Use the existing Django template to communicate that
             # back to the user.
             context = {
-                'validlink': False,
+                'validlink': True,
```

Comment:
@nasthagiri I git blamed, and saw it was `True` originally and changed to `False` later on. 
During the investigation, I saw the `valid link` means that the password reset link is valid or not. The link (code) get invalid (expire) when it is consumed. In this case, the link is not expired, means we can reuse that. 

We have show separate error msg when the link get invalid.
`This password reset link is invalid. It may have been used already. To reset your password, go to the sign-in page and select Forgot password`

So It made sense to me that shouldn't show the above message because the link is still valid.